src/main/scala/ai/privado/audit/DataElementDiscovery.scala

```diff
@@ -592,11 +592,14 @@ object DataElementDiscoveryJS {
             AuditReportConstants.AUDIT_EMPTY_CELL_VALUE
           )
           val ruleMemberInfo = taggedMemberInfo.getOrElse(key.fullName, new mutable.HashMap[String, String])
+          val addedMembers   = mutable.Set[String]()
+
           value.foreach {
             case (member: Member) => {
-              if (member.name.nonEmpty) {
+              val memberDetails = s"${key.fullName}${key.file.head.name}${member.name}"
```

Comment:
can we rename these memberDetails, localDetails, identifierDetails to a more descriptive one like memberUniqueKey